pkgs/top-level/aliases.nix

```diff
@@ -42,6 +42,7 @@ mapAliases ({
   aircrackng = aircrack-ng; # added 2016-01-14
   alienfx = throw "alienfx has been removed."; # added 2019-12-08
   ammonite-repl = ammonite; # added 2017-05-02
+  antimicro = throw "antimicro has been removed as it was broken, see antimicroX instead.";
```

Comment:
Sorry for nitpicking, could you also add the addition date, like the other entries?

Reply:
Yes of course, thank you for ensuring regularity :)